app/dlc-suredbits-client-test/src/test/scala/org/bitcoins/sbclient/MockServer.scala

```diff
@@ -0,0 +1,43 @@
+package org.bitcoins.sbclient
```

Comment:
same thing with this, we probably want to re-use this in testkit

Reply:
it is now pretty dlc client specific, but should another similar thing be needed I'm sure that some super type can be factored out into testkit